llvm/test/tools/llvm-readobj/ELF/Sparc/elf-headers.test

```diff
@@ -0,0 +1,78 @@
+## Test various combinations of ELF flag values for SPARC.
```

Comment:
Nit: since the spelling appears to be "SPARC", not "Sparc", I'd rename the directory accordingly.